roles/kubernetes/master/templates/kubeadm-config.v1beta2.yaml.j2

```diff
@@ -127,17 +175,17 @@ apiServer:
     feature-gates: {{ kube_feature_gates|join(',') }}
 {% endif %}
 {% if cloud_provider is defined and cloud_provider in ["openstack", "azure", "vsphere", "aws"] %}
-    cloud-provider: {{cloud_provider}}
-    cloud-config: {{ kube_config_dir }}/cloud_config
+    cloud-provider: {{ cloud_provider }}
+    cloud-config: {{ cloud_config_file }}
```

Comment:
I couldn't find `cloud_config_file` definition anywhere have I missed it?

Reply:
I'll move cloud config changes to a separate pr